packer/securing-docker/README.md

```diff
@@ -0,0 +1,68 @@
+
+To accomplish the building of this AMI we use [Packer](https://www.packer.io/), which is an easy way to automate the creation of your images.  It supports multiple providers (i.e. AWS, Digital Ocean) and allows you to document in a repo how your images were built and modifications that were done to them over time.
+
+The work to get this AMI passing the [Docker-Bench-Security](https://github.com/docker/docker-bench-security) was not a small task, but a critical task of any development organization. Minimizing the attack surface of an application we deploy into the wild will always return on investment. Whether that return comes from keeping an active attacker from escalating privileges and damaging our craft or simply letting you sleep at night knowning that you've done everything you can to secure your environments. Give it a spin and feel free to raise [issues or PRs](https://github.com/nearform/devops) if you find ways to improve upon the work.
+
+#### Requirements 
+
+- [Packer](https://www.packer.io/intro/getting-started/install.html)
+- [AWS account](https://aws.amazon.com/) - free tier will work with our examples.
+- [Security Credentials](https://console.aws.amazon.com/iam/home) from your AWS account
+
+#### Setup
+ 
+ 1. Setup packer
+ 2. `$ git clone git@github.com:nearform/devops.git`
+ 3. Create a file named `variables.json` in the root of the example repo from #2.  Add your access and secret keys. If you want to use a different region in AWS change the `aws_region` and find replace the `ubuntu_source_ami` with the AMI ID that has an instance type of `hvm-ssd` from [Ubuntu](https://cloud-images.ubuntu.com/locator/).  Your File should look something like this:
+ 
+``` json
+{
+  "aws_secret_key": "< INSERT YOUR SECRET KEY >",
+  "aws_access_key": "< INSERT YOUR ACCESS KEY >",
+  "aws_region": "us-east-1",
+  "ubuntu_source_ami": "ami-cb1d41b0"
+}
+```
+ 
+ #### Build the AMI
+ After executing the command below you should see an AMI in the [AWS Console](https://console.aws.amazon.com/ec2/v2/home?#Images)
+ 
+ `$ packer build -var-file=variables.json base-image-us-east-template.json`
+
+Our example template uses packer's [amazon-ebs](https://www.packer.io/docs/builders/amazon-ebs.html) builder.  We use a builder and two provisioners in our template to accomplish the following:
+
+1. Create /etc/docker/daemon.json
+2. Create and mount the volume that will be used to store our Docker assets
+3. Install Docker CE
+4. Clone [Docker-Bench-Security](https://github.com/docker/docker-bench-security) 
+5. Install and configure auditd
```

Comment:
People might not necessarily be familiar with what `auditd` is, perhaps linking to it might be useful